tests/gfapi-test.c

```diff
@@ -89,8 +89,8 @@ main(int argc, char *argv[])
 
 out:
   if (tgfd) {
-    glfs_unlink(fs, GB_TEST_FILE);
     glfs_close(tgfd);
+    glfs_unlink(fs, GB_TEST_FILE);
```

Comment:
thinking more about this, should this line move out of 'if()' condition check?

Reply:
IMO, No need and logic will be like:
```
 
tgfd = create();
if (!tgfd) {
     ....
    goto out;
}

tgfd = open();
if (!tgfd) {
    ....
    glfs_unlike();
    goto out;
}

out:
if (tgfd) {
    glfs_close();
    glfs_unlike();
}
```